### ISSUE TITLE
Add support for print Bible to pdf

### ISSUE THREAD

**Issue by markellishancock** [unfoldingWord-dev/ts-desktop#311]
David Reeves [04:27] 
We’ve run into an issue here in India with tS that I’m going to attempt to describe it here for the record and hope there are plans to address this in the near future. Here is what happened.

On the first day of the workshop began laptops were given out and instructions were given on how to set up tS for use in their respective languages. Teams were encouraged to try out the software. This was not serious work. Just dabbling around with the tool to see how it worked in a random section of scripture. The workshop progressed and the teams began to do serious translation work in their assigned sections. When the workshop ended everyone had uploaded their work. The last team to upload had dabbled in some other sections of scripture during the initial software introduction time. This bogus text then overwrote good text that had been drafted leaving a complex mess to unravel after the workshop had ended.

One solution would be to only allow teams to play around in a demo project but that won’t solve the issue of the last person to upload overwriting all earlier work. It needs to be very clear when you go to upload that you are uploading the correct text and not overwriting someone else’s materials with your junk. Perhaps that is the way it is designed to work but that is not what happened. We had a very complex mess to clean up.

Because of this the teams are concerned about using tS for any additional editing work beyond this current workshop. Is there something that was missed in the initial instructions that would have prevented this?

[04:30] 
I know someone will is going to tell me this can be fixed in door43 but these teams don’t all have internet access and for those who do we would be introducing them into the complex editing environment of door43. We’d like to be able to do everything in tS. Is that not possible?

David Reeves [03:39] 
I suspect that somewhere the tS development team has a list of features that they are working on. If this is not already on the list could you consider adding it. We need a “Save to” function in tS. The menu would include “Save to pdf”, “Save to text and/or rtf”.. We’ve been using the workaround of using a word processor in the workshop which gave us a printed copy of Acts at the end of this week but it did nothing to help the teams learn how to use tS. Now they also need to cut and paste 400 chunks of text from their Open Office word processor into tS. This process is S L O W. The "save to text” function would also allow the translation team leader to gather and combine translation text from various team members which are working from various locations and do not have internet access in their villages. We don’t want to repeat what we did this week. We need the teams working in tS only so they can master the tool while help is available to them.

**Comment by RRohrssen** [comment]
OBS is working and will be in the next release.

We will work on adding print capability for bible.

**Comment by a-gundy** [comment]
Regarding @tjklemz comment in issue #220, should I work on this using PDFkit or should I wait to see what the decision is on using Electron's PDF library?

**Comment by neutrinog** [comment]
I re-opened that issue for discussion. Good question @a-gundy. Let's figure that out.

**Comment by neutrinog** [comment]
For now we will go forward with the current pdf library. Later we will migrate to using the pdf printer provided by electron.

**Comment by cckozie** [comment]
Testing on OSX, tS 6.0, OBS project
I did NOT translate the project title so it was blank.
I translated the first story title and story reference chunks, but did not mark them as complete.
I translated the first story's first chunk and marked it as complete.
I did a Print/Save to PDF with all of the boxes unchecked.
The output file does not contain the translated story's first chunk (pdf attached)
[en-x-demo1_obs_text_obs.pdf](https://github.com/unfoldingWord-dev/ts-desktop/files/204901/en-x-demo1_obs_text_obs.pdf)
I then went back and translated the project title, reprinted, and it worked as expected.
